### PR TITLE
Add Sales Graph dashboard widget

### PR DESCRIPTION
**Pull request by drbyte**

### CODE REVIEW

File: admin/includes/template/dashboardWidgets/tplSalesGraphReport.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ *
+ * @package templateSystem
+ * @copyright Copyright 2003-2016 Zen Cart Development Team
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: $
+ */
+?>
+<div class="row">
+  <div class="col-xs-12">Monthly Sales (excludes shipping)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<?php echo '<a href="' . zen_href_link(FILENAME_STATS_SALES_REPORT_GRAPHS) . '">Click here for complete details...</a>'; ?></div>
+</div>
```
Comment (scottcwilson) [inline admin/includes/template/dashboardWidgets/tplSalesGraphReport.php]: Move strings to language file perhaps?

**Comment by drbyte** [comment]
> Move strings to language file perhaps?

Okay, ya. But, two things on that:
1. The graph itself isn't multilanguage friendly
2. Growing the dashboard_widgets.php language file is going to be a recipe for disaster. Needs a better solution.

**Comment by zcwilt** [comment]
:+1:

**Comment by ajeh** [comment]
:+1:

**Pull request merged**